lib/path.js

```diff
@@ -115,8 +138,8 @@ win32.resolve = function() {
 
     var result = splitDeviceRe.exec(path),
         device = result[1] || '',
-        isUnc = device && device.charAt(1) !== ':',
-        isAbsolute = win32.isAbsolute(path),
+        isUnc = device && device[1] !== ':',
+        isAbsolute = isUnc || result[2],
```

Comment:
I'm not a big fan on inlining the call to `win32.isAbsolute` here and in `win32.normalize`. I'm concerned that we could get to a point that `win32.isAbsolute` and these inline versions diverge.